### ISSUE TITLE
AOT compilation fails

### ISSUE THREAD

**Issue by albpara** [matthias-schuetz/angular2-seed#15]
I am getting this error:

`Error: Error at /foo/bar/app/src/bootstrap.aot.ts:2:36: Cannot find module '../../app-aot/app/src/app.module.ngfactory'.`

I noticed that only a `node_modules` foldes appeared under `app-aot`, tehere is no app folder inside, so basically the app.module.ngFactory is not being generated but dunno why

Any clue about the problem?

**Comment by matthias-schuetz** [comment]
It seems there's something wrong with the AoT configuration. If you played around with the seed structure, you'll need to adjust the `aot-rollup-config.js` and `tsconfig-aot.json` (the `files` array).
I can't give a direct advice but changing the project file structure means adjusting the configuration files.

**Comment by albpara** [comment]
That is weird, I tried the same with a different project where I use the very same structure and the one on the seed and I am getting the same error.

No app.module.ngFactory was generated

**Comment by albpara** [comment]
I notice that this problem appears after updating to typescript 2.4.2

**Comment by matthias-schuetz** [comment]
Yes, updating TypeScript will cause that issue. I set the TypeScript version to a specific one in the `package.json` to prevent such problems since I can't control what happens to the seed when TypeScript gets updated.

Unfortunately I can't tell which TypeScript version is the latest one that works with this seed but in order to fix the problem, all dependencies of the seed need to be updated and re-tested. I can't tell when I'll have the time to do this but TypeScript is definitely the issue here.
